### PR TITLE
Adding Travis

### PR DESCRIPTION
**Pull request by afshinm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  - "node"
+
+install:
+  - npm install
+
+script:
+  - make test
```
Comment (bozdoz) [inline .travis.yml]: why not just `npm test`?  Not really sure why we even need to keep the makefile.  But that's purely opinion.  Think it makes sense to have the commands in one place.  Also think npm is standard.

Comment (afshinm) [reply .travis.yml]: yeah sure, that works, too. I think we can remove `script`. I just thought `make test` is more standard :D

Comment (bozdoz) [reply .travis.yml]: Yeah maybe you're right; guess travis doesn't require much.  This is from another repo I've worked on:

https://github.com/ursudio/leaflet-webgl-heatmap/blob/master/.travis.yml#L1-L3

It's just:

```
language: node_js
node_js:
  - node # latest node.js version
```

automatically does `npm install` and `npm test`

Comment (afshinm) [reply .travis.yml]: @bozdoz so removing the extra parts?

**Comment by afshinm** [comment]
closes #785

**Pull request merged**